Dockerfile

```diff
@@ -28,5 +29,26 @@ COPY entrypoint.sh /usr/local/bin/
 
 VOLUME ["$DATA_DIR"]
 
+#for linux-headers
+WORKDIR /kernel
+COPY  --from=ksrc /kernel-dev.tar .
+RUN tar xf kernel-dev.tar
+
+#bpftrace
+ENV BPFTRACE_VERSION v0.15.0
+WORKDIR /bpftrace
+RUN wget https://github.com/iovisor/bpftrace/releases/download/${BPFTRACE_VERSION}/bpftrace && \
+    chmod +x bpftrace && \
+    mv bpftrace /bin && \
+    mkdir -p /tracing && \
+    wget https://github.com/iovisor/bpftrace/archive/${BPFTRACE_VERSION}.zip && \
+    unzip ${BPFTRACE_VERSION}.zip && \
+    cp -r bpftrace*/tools /tracing &&\
+    mv /kernel/usr/src/linux-headers* /kernel/usr/src/linux-headers
```

Comment:
Why not just install bpftrace with apt in tapyrus/builder?

Reply:
done.